src/sonic-yang-models/yang-models/sonic-smart-switch.yang

```diff
@@ -157,6 +160,149 @@ module sonic-smart-switch {
 			}
 		}
 		/* end of container DPU */
+
+		container REMOTE_DPU {
+			description "REMOTE_DPU part of config_db.json";
+
+			list REMOTE_DPU_LIST {
+				key "dpu_name";
+
+				leaf dpu_name {
+					description "Name of the dpu";
+					type string {
+						pattern "[a-zA-Z0-9-]+[0-9]+";
+					}
+				}
+
+				leaf type {
+					description "Type of dpu";
+					type string;
+				}
+
+				leaf pa_ipv4 {
+					description "DPU ipv4 physical address";
+					type inet:ipv4-address;
+				}
+
+				leaf pa_ipv6 {
+					description "DPU ipv6 physical address";
+					type inet:ipv6-address;
+				}
+
+				leaf npu_ipv4 {
+					description "Loopback ipv4 of remote NPU";
+					type inet:ipv4-address;
+				}
+
+				leaf npu_ipv6 {
+					description "Loopback ipv6 of remote NPU";
+					type inet:ipv6-address;
+				}
+
+				leaf dpu_id {
+					description "ID of the DPU from minigraph";
+					type string {
+						pattern [0-7];
+					}
+				}
+
+				leaf swbus_port {
+					description "TCP listening port for swbus service for this DPU. Must be 23606 + dpu_id";
+					type inet:port-number;
+					must "current() = 23606 + number(current()/../dpu_id)";
+				}
+			}
+		}
+		/* end of container REMOTE_DPU */
+
+		container VDPU {
+			description "VDPU part of config_db.json";
+
+			list VDPU_LIST {
+				key vdpu_id;
+
+				leaf vdpu_id {
+					description "ID of the VDPU";
+					type string {
+						pattern "vdpu[0-9]+";
+					}
+				}
+
+				leaf profile {
+					description "VDPU profile. Currently unused, reserved for future use.";
+					type string;
+				}
+
+				leaf tier {
+					description "VDPU tier. Currently unused, reserved for future use.";
+					type string;
+				}
+
+				leaf-list main_dpu_ids {
+					description "Main DPUs involved in this VDPU";
+					type string {
+						pattern "[a-zA-Z0-9-]+[0-9]+";
+					}
+				}
+			}
+		}
+		/* end of container VDPU */
+
+		container DASH_HA_GLOBAL_CONFIG {
+			description "DASH_HA_GLOBAL_CONFIG part of config_db.json";
+
+			list DASH_HA_GLOBAL_CONFIG_LIST {
```

Comment:
I don't think it should be a list, since all entry are only allow "global"

Reply:
Got it.